lib/frr_pthread.c

```diff
@@ -182,3 +183,8 @@ unsigned int frr_pthread_get_id()
 {
 	return next_id++;
 }
+
+void frr_pthread_yield(void)
+{
+	(void)sched_yield();
```

Comment:
I thought sched_yield didn't work predictably for all schedulers/kernels and that it was safer to use sleep(0).  Is this dated (or wrong) info?